Projects/l0/Sniffer/lib/Sniffer/sniffer.h

```diff
@@ -0,0 +1,25 @@
+/******************************************************************************
+ * @file sniffer
+ * @brief monitoring tool that receives and transmits serially all msgs
+ * @author Luos
+ * @version 0.0.0
+ ******************************************************************************/
+
+#include "luos.h"
+#include "cmd.h"
+#include "streaming.h"
+#include "reception.h"
+/*******************************************************************************
+ * Definitions
+ ******************************************************************************/
+
+/*******************************************************************************
+ * Variables
+ ******************************************************************************/
+
+/*******************************************************************************
+ * Function
+ ******************************************************************************/
+void 				Sniffer_Init(void);
+void 				Sniffer_Loop(void);
+streaming_channel_t *get_Sniffer_StreamChannel();
```

Comment:
This .h file is the interface used on a package level (on the main).
by adding `get_Sniffer_StreamChannel` function on it you say that this function is for the main.
As I understand it this function is published because you need it into sniffer_com.c and sniffer_data.c. The good way in my point of view is to declare this function in another .h.
So you should declare it into sniffer_data.h and include sniffer_data.h on sniffer.c